chain/src/Pos/Chain/Block/Main/Types.hs

```diff
@@ -54,7 +54,7 @@ instance Bi MainProof where
                          decode <*>
                          decode
 
-instance SafeCopy SscProof => SafeCopy MainProof where
```

Comment:
Was this a redundant constraint or something? Just curious.

Reply:
The old code base used these to side step orphan instance problems.